platform/viewer/src/routes/WorkList/StudyListConfig.jsx

```diff
@@ -0,0 +1,218 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Added a new file to specify the study list portion of the worklist.  It is configurable by modifying the config-point stuff to do things like adding or removing coluumns, defining new columns etc.